Sources/RInAppMessaging/Services/ImpressionService.swift

```diff
@@ -53,7 +47,7 @@ internal class ImpressionService: ImpressionServiceType, HttpRequestable, TaskSc
         // Broadcast impression data to RAnalytics.
         // `impression` type is sent separately, just after campaign is displayed.
         let impressionData = impressions.filter({ $0.type != .impression }).encodeForAnalytics()
-        AnalyticsBroadcaster.sendEventName(Constants.RAnalytics.impressionsEventName,
+        AnalyticsTracker.sendEventName(Constants.RAnalytics.impressionsEventName,
                                            dataObject: [Constants.RAnalytics.Keys.impressions: impressionData,
```

Comment:
nit: please fix indentation in `dataObject` lines

Reply:
Fixed